packages/genui/openui/src/core/renderer.css

```
@@ -509,6 +523,24 @@
   border-color: rgba(59, 130, 246, 1);
 }
 
+.OpenUICheckBoxMark {
+  color: #ffffff;
+  font-size: 24rpx;
```

Stylelint (17.13.0)

[error] Unknown unit "rpx" (unit-no-unknown)

(unit-no-unknown)